### PR TITLE
Create wai-aria/role/tab-roles.html

### PR DESCRIPTION
**Pull request by rahimabdi**
Closes https://github.com/web-platform-tests/interop-2023-accessibility-testing/issues/17

### CODE REVIEW

File: wai-aria/role/tab-roles.html [hunk 1]
```diff
@@ -0,0 +1,30 @@
...
+    <script src="/resources/testdriver-vendor.js"></script>
+    <script src="/resources/testdriver-actions.js"></script>
+    <script src="/wai-aria/scripts/aria-utils.js"></script>
+</head>
+<body>
+
+<p>Tests <a href="https://w3c.github.io/aria/#tab">tab</a> and related roles.</p>
+
+<div role="tablist" data-testname="div role is tablist" data-expectedrole="tablist" class="ex">
+    <div role="tab" data-testname="role is tab (in div tablist)" data-expectedrole="tab" class="ex">x</div>
```
Comment (cookiecrook) [inline wai-aria/role/tab-roles.html:17]: I think this should include some selected and unselected tabs too... I don't anticipate that will change the result, but good to have a test that confirms it.

Comment (cookiecrook) [reply wai-aria/role/tab-roles.html:17]: likewise a focusable versus non-focusable... maybe a static "roving" tab index... No need to hook up the JS, but something like `aria-selected="true" tabindex="0"` on the selected tab, and `tabindex="-1"` on the unselected tabs.

File: wai-aria/role/tab-roles.html [hunk 2]
```diff
@@ -0,0 +1,30 @@
...
+</div>
+<div role="tabpanel" data-testname="role is tabpanel (with sibling div tablist)" data-expectedrole="tabpanel" class="ex"></div>
```
Comment (cookiecrook) [inline wai-aria/role/tab-roles.html:19]: Might be good to have a second tabpanel (after the next tablist) that includes some DOM content in the element.

File: wai-aria/role/tab-roles.html [hunk 3]
```diff
@@ -0,0 +1,30 @@
...
+
+<div role="tablist" data-testname="div role for button parent container is tablist" data-expectedrole="tablist" class="ex">
+    <button role="tab" data-testname="button role is tab (in div tablist)" data-expectedrole="tab" class="ex">x</div>
```
Comment (cookiecrook) [inline wai-aria/role/tab-roles.html:22]: Good idea to switch up the button to an interactive... Maybe a `ul`/`li` test, too?

File: wai-aria/role/tab-roles.html (hunk 1 above)
Comment (rahimabdi) [reply wai-aria/role/tab-roles.html:18]: Added examples with selected/unselect tabs, and several roving tabindex examples.

File: wai-aria/role/tab-roles.html (hunk 2 above)
Comment (rahimabdi) [reply wai-aria/role/tab-roles.html:20]: Sure thing; added several examples with non-empty tabpanel content.

File: wai-aria/role/tab-roles.html (hunk 3 above)
Comment (rahimabdi) [reply wai-aria/role/tab-roles.html:58]: Done; also added a `ul`/`li` example that implements role="none" on the `li` elements

**Review by ASISBusiness — APPROVED** [review]

File: wai-aria/role/tab-roles.html [hunk 4]
```diff
@@ -0,0 +1,90 @@
...
+    <div role="tab" data-testname="role is tab (in div tablist)" data-expectedrole="tab" class="ex">x</div>
+</div>
+<div role="tabpanel" data-testname="role is tabpanel (with sibling div tablist)" data-expectedrole="tabpanel" class="ex"></div>
+
+<!--<div> tabs with tablist/tab/tabpanel semantics, selection, no focus management-->
+<div role="tablist" data-testname="div role is tablist (with selection)" data-expectedrole="tablist" class="ex">
+    <div role="tab" aria-selected="true" data-testname="role is tab and tab is selected" data-expectedrole="tab" class="ex">x</div>
+    <div role="tab" aria-selected="false" data-testname="role is tab and tab is not selected" data-expectedrole="tab" class="ex">y</div>
+</div>
+<div role="tabpanel" data-testname="role is tabpanel (faux selected)" data-expectedrole="tabpanel" class="ex"></div>
```
Comment (cookiecrook) [inline wai-aria/role/tab-roles.html:27]: What makes this tabpanel "faux selected"?

**Review by cookiecrook — APPROVED** [review]
Very cool that all engines pass all the tests... That might be a first!

File: wai-aria/role/tab-roles.html (hunk 4 above)
Comment (rahimabdi) [reply wai-aria/role/tab-roles.html:27]: @cookiecrook Just ensuring that test names are globally unique!

**Pull request merged**